tutorials/polkadot-sdk/parachains/local-chain/spin-your-nodes.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Add Trusted Nodes
+title: Spin Your Nodes
```

Comment:
spin up?

Reply:
Agree I'll change this for Spin Up Your Nodes